boards/arm/nucleo_f334r8/doc/nucleof334r8.rst

```diff
@@ -0,0 +1,182 @@
+.. _nucleo_f334r8_board:
+
+ST Nucleo F334R8
+################
+
+Overview
+********
+STM32 Nucleo-64 development board with STM32F334R8 MCU, supports Arduino and ST morpho connectivity.
+
+The STM32 Nucleo board provides an affordable and flexible way for users to try out new concepts
+and build prototypes with the STM32 microcontroller, choosing from the various
+combinations of performance, power consumption and features. 
+
+The Arduino™Uno V3 connectivity support and the ST morpho headers allow to expand easily 
```

Comment:
Since Arduino  is a third-party trademark we just use a asterisk (*) and put a space after it